e2store/src/bin/era2-stats.rs

```diff
@@ -23,12 +24,12 @@ struct Stats {
     storage_item_count: usize,
 }
 
-/// Reads the era2 file and prints stats about it.
+/// Reads the era2 file, validates the format, and prints stats about it.
 ///
 /// It can be run with following command:
 ///
 /// ```bash
-/// cargo run -p e2store --bin era2-stats --features era2-stats-binary -- <path>
+/// cargo run --release -p e2store --bin era2-stats --features era2-stats-binary -- <path>
```

Comment:
I assume this is just because the operation is slow, so `--release` helps speed it up?

Reply:
Yes, that's correct.